extensions/amp-base-carousel/0.1/test-e2e/test-multi-visible-single-advance.js

```diff
@@ -45,15 +44,15 @@ describes.endtoend(
       const slides = await getSlides(controller);
 
       // Click `next` more than necessary
-      for (let i = 0; i < 7; i++) {
+      for (let i = 0; i < slideCount + 2; i++) {
```

Comment:
Why `+ 2`? Isn't clicking by `slideCount` already sufficiently more than necessary?

Reply:
It should be, but I'd like the extra insurance that we're actually testing more "next" clicks than necessary, since that is how the bug originally manifested.